LeetCodeAssistant/Utility.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+extension String {
+    func removeHtmlTag() -> String {
+        return replacingOccurrences(of: "<(\"[^\"]*\"|'[^']*'|[^'\">])*>", with: "", options: .regularExpression, range: self.range(of: self)).replacingOccurrences(of: "&quot;", with: "").replacingOccurrences(of: "&nbsp;", with: "").replacingOccurrences(of: "&#39;", with: "'")
+    }
+    
+    
+}
```

Comment:
⚠️ Extensions with no restriction affect for all String values in whole the app. This is a widely known anti-pattern because it may causes unexpected behavior. If you wanna extend String, you should put this extension only in the file you use and put `fileprivate`.

Reply:
I guess there's no usage for raw HTML tags. Can you hold just parsed content string instead of raw HTML if you don't think of using raw HTML?